modules/src/main/modules/search/elasticsearch/module.xml

```diff
@@ -8,17 +8,13 @@
 <module xmlns="urn:jboss:module:1.1" name="org.hibernate.search.elasticsearch" slot="${hibernate.search.version}">
     <resources>
         <resource-root path="hibernate-search-elasticsearch-${hibernate.search.version}.jar" />
-        <resource-root path="rest-${elasticsearchClientVersion}.jar" />
-        <resource-root path="sniffer-${elasticsearchClientVersion}.jar" />
-        <resource-root path="gson-${elasticsearchGsonVersion}.jar" />
     </resources>
     <dependencies>
-        <module name="org.hibernate.search.engine" slot="${hibernate.search.version}" />
-        <module name="org.hibernate.commons-annotations" />
+        <module name="org.hibernate.search.engine" slot="${hibernate.search.version}"/>
+        <module name="org.hibernate.search.elasticsearch-client" slot="${hibernate.search.version}"/>
         <module name="javax.xml.bind.api"/>
```

Comment:
I don't think we depend on `javax.xml.bind.api`, only the rest client does (the sniffer, actually).